src/components/language-switch/language-switch.tsx

```diff
@@ -1,44 +1,91 @@
 import { useI18next } from 'gatsby-plugin-react-i18next';
 import React from 'react';
 import styled from 'styled-components';
-import { Link } from '../links/links';
+import { Chevron } from '../icons/chevron';
 
-const StyledLanguageLink = styled(Link)<{ selected?: boolean }>`
-  margin: 1px 0 1px 12px;
+interface LanguageSwitchProps {
+  translation: any;
+  className?: string;
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}
+
+const StyledNav = styled.nav<{
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}>`
+  transition: color 0.2s;
+  color: ${(props) =>
+    props.fromNavigation || props.$lightTheme
+      ? props.theme.palette.text.default
+      : props.theme.palette.text.header};
+  &:hover {
+    color: ${(props) =>
+      props.fromNavigation
+        ? props.theme.palette.text.header
+        : !props.$lightTheme &&
+          !props.fromNavigation &&
+          props.theme.palette.text.headerHover};
+  }
+`;
+
+const StyledSelection = styled.select<{
```

Comment:
I like the select/dropdown but it shows only `de` and `en` while we have no the space to write `Deutsch` and `English` and such (which was a problem before).

Can you change the labels in the dropdown? I think we should have the full names laying around somewhere already? 
![Screen Shot 2021-11-08 at 10 33 22](https://user-images.githubusercontent.com/1701755/140717922-1b9a6f1c-6aa6-46fa-85ca-0f3b748590b1.png)

Reply:
Do you know if there is a pure HTML/CSS solution for displaying the abbreviation (e.g. "EN") when collapsed and the full text (e.g. "English") when expanded?

I talked briefly with Fabian about this and in our opinion, there is only the possibility to build a custom dropdown component to achieve this behavior.

Reply:
This is not possible with the native `select` element. I'm pretty sure this would require a custom solution. I'd say that this is out of scope of this PR. I suggest we iterate on the LanguageSwitcher at a later point and merge this with the simple native `de/en` dropdown.

Reply:
out of scope, I agree. I will remove this from my list of expectations then.